src/mqtt.cpp

```diff
@@ -294,9 +413,36 @@ mqtt_loop() {
     }
   }
 
+  if((millis() - mqttStatusLoopTime > MQTT_STATUS_LOOP_TIME) || !mqttStatusLoopTime ) {
+    mqttStatusLoopTime = millis(); // reset
+    mqtt_pub_status();
+  }
```

Comment:
These values are evented as fast as they are read from the EVSE module so I don't think running a faster loop here will get you anything.

If you need faster updates, maybe the polling time can just be made configurable? but I think that should be a separate PR

Reply:
It does. I can see fluctating values each 2 sec for live Amp , and pilot get refreshed in time. If not doing that it can takes 30 sec. 
You can see I only push few essential values in this loop to not spam the mqtt server. The others are refreshed by the main loop. I would prefer those to be pushed directly by the EVSE module when value has changed, but for now I haven't found a better way to do without modifications on the EVSE firmware.

I can make the poilling configurable in the UI

Reply:
Seems I have a better way of doing this, instead of polling, can I for example, add a send_event in the EvseMonitor::setPilot, just after we receive the RAPI_RESPONSE_OK ?

Reply:
Hum... Oh right yeah some of those values are read more frequently... Yeah I guess we can do that. Can always make it configurable  if people complain